src/composeinfo.h

```diff
@@ -0,0 +1,24 @@
+#ifndef AKTUALIZR_LITE_COMPOSEINFO_H
+#define AKTUALIZR_LITE_COMPOSEINFO_H
+
+#include <json/json.h>
+#include <string>
+#include <vector>
+#include "yaml2json.h"
+namespace Docker {
+
+class ComposeInfo {
+ public:
+  ComposeInfo(const std::string& yaml);
+  bool getServices(std::vector<Json::Value>& services);
```

Comment:
Any reason to not convert to actual types? e.g
~~~
struct ComposeService {
  std::string name
  std::string svc_hash;
};
~~~

Then you could drop the 2 helper methods below.

Reply:
We agreed to work on it iteratively.